packages/@glimmer/runtime/lib/vm/append.ts

```diff
@@ -27,6 +27,11 @@ export interface PublicVM {
   newDestroyable(d: Destroyable): void;
 }
 
+export interface IteratorResult<T> {
```

Comment:
I don't think so or at least I couldn't find any docs on it. https://www.typescriptlang.org/docs/handbook/iterators-and-generators.html